modules/csg/csg.cpp

```diff
@@ -1068,6 +1068,8 @@ void CSGBrushOperation::Build2DFaces::_merge_faces(const Vector<int> &p_segment_
 }
 
 void CSGBrushOperation::Build2DFaces::_find_edge_intersections(const Vector2 p_segment_points[2], Vector<int> &r_segment_indices) {
+	Vector<Vector<Vector2>> processed_edges;
```

Comment:
Try LocalVector

Reply:
Thanks, works fine here. See the other comment for complications with the other one